command/job_plan.go

```diff
@@ -153,13 +154,49 @@ func (c *JobPlanCommand) Run(args []string) int {
 		opts.PolicyOverride = true
 	}
 
+	if job.IsMultiregion() {
+		return c.multiregionPlan(client, job, opts, diff, verbose)
+	}
+
 	// Submit the job
 	resp, _, err := client.Jobs().PlanOpts(job, opts, nil)
 	if err != nil {
 		c.Ui.Error(fmt.Sprintf("Error during plan: %s", err))
 		return 255
 	}
 
+	exitCode := c.formatPlannedJob(job, resp, diff, verbose)
+	c.Ui.Output(c.Colorize().Color(formatJobModifyIndex(resp.JobModifyIndex, path)))
+	return exitCode
+}
+
+func (c *JobPlanCommand) multiregionPlan(client *api.Client, job *api.Job, opts *api.PlanOptions, diff, verbose bool) int {
+
+	var exitCode int
+
+	for _, region := range job.Multiregion.Regions {
+		regionName := region.Name
+		client.SetRegion(region.Name)
+
+		// Submit the job for this region
+		resp, _, err := client.Jobs().PlanOpts(job, opts, nil)
+		if err != nil {
+			c.Ui.Error(fmt.Sprintf("Error during plan for region %q: %s", regionName, err))
+			return 255
```

Comment:
Would it make sense to keep going and report all failed regions at the end?  Don't know about the context here, but I find it helpful when I get multiple errors together rather than one error at a time.

Reply:
That sounds like a better option!